### PR TITLE
[PL-51736] Delegate 83200 Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: Adding Delegate release notes for DEL 83200
* Jira/GitHub Issue numbers (if any): [PL-51736](https://harness.atlassian.net/browse/PL-51736)
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

[PL-51736]: https://harness.atlassian.net/browse/PL-51736?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ For more information, go to [Delegate expiration support policy](/docs/platform/
 
 ## June 2024
 
+### Version 24.06.83200 <!--  May 8, 2024 -->
+
+- Delegate logs were displaying entire bearer tokens when using the IDP connector. Added log sanitization to delegate logs to mask commonly used secret patterns. These patterns can be extended on a per-use-case basis by adding them to the /opt/harness-delegate/sanitize-patterns.txt file inside the delegate (PL-47914) (PL-50889, ZD-64069).
```
Comment (brian-f-harness) [inline release-notes/delegate.md:54]:
```suggestion
- Delegate logs were displaying entire bearer tokens when using the IDP connector. Added log sanitization to delegate logs to mask commonly used secret patterns. These patterns can be extended on a per-use-case basis by adding them to the `/opt/harness-delegate/sanitize-patterns.txt` file inside the delegate. (PL-47914) (PL-50889, ZD-64069)
```

File: release-notes/delegate.md [hunk 2]
```diff
@@ -49,6 +49,10 @@ For more information, go to [Delegate expiration support policy](/docs/platform/
 
 ## June 2024
 
+### Version 24.06.83200 <!--  May 8, 2024 -->
+
+- Delegate logs were displaying entire bearer tokens when using the IDP connector. Added log sanitization to delegate logs to mask commonly used secret patterns. These patterns can be extended on a per-use-case basis by adding them to the `/opt/harness-delegate/sanitize-patterns.txt` file inside the delegate. (PL-47914) (PL-50889, ZD-64069)
```
Comment (brian-f-harness) [inline release-notes/delegate.md:54]:
```suggestion
- Delegate logs were displaying entire bearer tokens when using the IDP connector. Added log sanitization to delegate logs to mask commonly used secret patterns. These patterns can be extended on a per-use-case basis by adding them to the `/opt/harness-delegate/sanitize-patterns.txt` file inside the delegate. (PL-47914, PL-50889, ZD-64069)
```

**Review by brian-f-harness — APPROVED** [review]

File: release-notes/delegate.md [hunk 3]
```diff
@@ -49,6 +49,10 @@ For more information, go to [Delegate expiration support policy](/docs/platform/
 
 ## June 2024
 
+### Version 24.06.83200 <!--  May 8, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/delegate.md:52]:
```suggestion
### Version 24.06.83202 <!--  May 8, 2024 -->
```

Comment (brian-f-harness) [reply release-notes/delegate.md:52]: delegate version updated to 83002
https://harness.slack.com/archives/C011SD1S1U3/p1717757780207839

**Review by brian-f-harness — COMMENTED** [review]
delegate version updated to 83002
https://harness.slack.com/archives/C011SD1S1U3/p1717757780207839

please add to the delegate supported versions topic

File: release-notes/delegate.md [hunk 4]
```diff
@@ -49,6 +49,10 @@ For more information, go to [Delegate expiration support policy](/docs/platform/
 
 ## June 2024
 
+### Version 24.06.83202 <!--  May 8, 2024 -->
+
+- Delegate logs were displaying entire bearer tokens when using the IDP connector. Added log sanitization to delegate logs to mask commonly used secret patterns. These patterns can be extended on a per-use-case basis by adding them to the `/opt/harness-delegate/sanitize-patterns.txt` file inside the delegate. (PL-47914, PL-50889, ZD-64069)
```
Comment (pratmit) [inline release-notes/delegate.md:54]: What is the IDP connector? Can we expand it?

**Review by pratmit — COMMENTED** [review]
Added a comment

File: release-notes/delegate.md [hunk 5]
```diff
@@ -49,6 +49,10 @@ For more information, go to [Delegate expiration support policy](/docs/platform/
 
 ## June 2024
 
+### Version 24.06.83202 <!--  May 8, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/delegate.md:52]: The delegate image hasn't been released yet.
https://hub.docker.com/r/harness/delegate/tags

Comment (brian-f-harness) [inline release-notes/delegate.md:52]:
```suggestion
### Version 24.06.83202 <!--  June 10, 2024 -->
```

File: release-notes/delegate.md [hunk 6]
```diff
@@ -49,6 +49,10 @@ For more information, go to [Delegate expiration support policy](/docs/platform/
...
+
+- Delegate logs were displaying entire bearer tokens when using the IDP Kubernetes connector. Added log sanitization to delegate logs to mask commonly used secret patterns. These patterns can be extended on a per-use-case basis by adding them to the `/opt/harness-delegate/sanitize-patterns.txt` file inside the delegate. (PL-50889, ZD-64069)
```
Comment (brian-f-harness) [inline release-notes/delegate.md:54]:
```suggestion
- Delegate logs were displaying entire bearer tokens when using the IDP Kubernetes connector. Added log sanitization to delegate logs to mask commonly used secret patterns. These patterns can be extended per-use-case by adding them to the `/opt/harness-delegate/sanitize-patterns.txt` file inside the delegate. (PL-50889, ZD-64069)
```

**Review by brian-f-harness — APPROVED** [review]

File: release-notes/delegate.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 title: Delegate release notes
 sidebar_label: Delegate
 tags: [NextGen, "Delegate"]
-date: 2024-06-03T10:00
+date: 2024-06-10T10:00
```
Comment (brian-f-harness) [inline release-notes/delegate.md:5]:
```suggestion
date: 2024-06-11T10:00
```

File: release-notes/delegate.md [hunk 8]
```diff
@@ -49,6 +49,10 @@ For more information, go to [Delegate expiration support policy](/docs/platform/
 
 ## June 2024
 
+### Version 24.06.83202 <!--  June 10, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/delegate.md:52]:
```suggestion
### Version 24.06.83202 <!--  June 11, 2024 -->
```

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**